atlas_densities/densities/inhibitory_neuron_densities_optimization.py

```diff
@@ -491,11 +492,29 @@ def _check_variables_consistency(
             if np.isfinite(deltas.loc[region_name, cell_type]):
                 for desc_id in id_set:
                     if np.isnan(x_result.loc[desc_id, cell_type]):
-                        raise AtlasDensitiesError(
-                            f"Cell count estimate of region named '{region_name}' for cell type "
-                            f"{cell_type} was given for certain whereas the cell count of "
-                            f"descendant id {desc_id} is not certain."
-                        )
+                        if ((deltas.loc[region_name, cell_type] in [0.0, np.inf]) and  
```

Comment:
Here we have an
```
if (A and
    B or C
    ):
```
Can we add parentheses around the innermost condition (`(A and B)` or `(B or C)`)to make the code more readable?

Reply:
Ah! It is 
if (
[A or A] and (B or C)
)
Yes we can add more parentheses !

Reply:
**Small note** (sorry if it was trivial to you): the way this was written is because something that yields `nans != np.nan` so I couldn't use `[A or A] and [B or B]` I had to use `np.isnan()` for the second condition.